static/sw.js

```diff
@@ -0,0 +1,10 @@
+importScripts('/bower_components/sw-toolbox/sw-toolbox.js');
+
+toolbox.options.debug = true;
+toolbox.router.default = toolbox.fastest;
```

Comment:
Good point about the max-age, we haven't really looked in the cache header. I've created #226 to fix this.